tests/operators/test_agnostic_boolean_check.py

```diff
@@ -53,194 +53,114 @@ def drop_table_snowflake(
-
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.table = "boolean_check_test"
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                cls.table,
-                conn_id="postgres_conn",
-                database="pagila",
-                schema="public",
-            ),
-        ).operator.execute({"run_id": "foo"})
-
-        cls.snowflake_table = get_table_name("boolean_check_test")
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                conn_id="snowflake_conn",
-                table_name=cls.snowflake_table,
-                schema=os.getenv("SNOWFLAKE_SCHEMA"),
-                database=os.getenv("SNOWFLAKE_DATABASE"),
-                warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
-            ),
-        ).operator.execute({"run_id": "foo"})
-
-        cls.table = "boolean_check_test"
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                cls.table,
-                conn_id="bigquery",
-                schema=SCHEMA,
-            ),
-        ).operator.execute({"run_id": "foo"})
-        cls.table_sqlite = Table(
-            "boolean_check_test", conn_id="sqlite_conn", schema="sqlite_schema"
-        )
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=cls.table_sqlite,
-        ).operator.execute({"run_id": "foo"})
-
-    @classmethod
-    def tearDownClass(cls):
-        drop_table_snowflake(
-            table_name=cls.snowflake_table,
-            schema=os.getenv("SNOWFLAKE_SCHEMA"),
-            database=os.getenv("SNOWFLAKE_DATABASE"),
-            warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
-            conn_id="snowflake_conn",
-        )
+# DROP TABLE IF EXISTS {table_name} CASCADE;
 
-    def clear_run(self):
-        self.run = False
-
-    def setUp(self):
-        super().setUp()
-        self.clear_run()
-        self.addCleanup(self.clear_run)
-        self.dag = DAG(
-            "test_dag",
-            default_args={
-                "owner": "airflow",
-                "start_date": DEFAULT_DATE,
-            },
-        )
 
-    def test_happyflow_postgres_success(self):
-        a = boolean_check(
-            table=Table(
-                self.table,
-                database="pagila",
-                conn_id="postgres_conn",
-            ),
-            checks=[Check("test_1", "{{table}}.rooms > 3")],
+@pytest.fixture(scope="module")
+def table(request):
+    boolean_check_table = Table(
+        "boolean_check_test",
+        database="pagila",
+        conn_id="postgres_conn",
+        schema="airflow_test_dag",
+    )
+    boolean_check_table_bigquery = Table(
+        "boolean_check_test",
+        conn_id="bigquery",
+        schema=SCHEMA,
+    )
```

Comment:
What happens if a user who doesn't have bigquery credentials tries to run this test?